### PR TITLE
Gj debug turn

### PR DESCRIPTION
**Pull request by grace-joh**
## Describe your changes
- fixed require statement in Computer
- refactored loop in Turn #find_valid_spot

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] I have added/updated corresponding tests.

### CODE REVIEW

File: lib/computer.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-require './player'
+require './lib/player'
```
Comment (margdaly) [inline lib/computer.rb:1]: Thank you for fixing this!

File: lib/turn.rb [hunk 2]
```diff
@@ -21,8 +21,11 @@ def move(player)
   end
 
   def find_valid_space(player)
-    input = player.symbol == 'X' ? $stdin.gets.chomp.upcase : player.input
-    find_valid_space(player) unless valid?(input)
+    input = nil
+    loop do
+      input = player.symbol == 'X' ? $stdin.gets.chomp.upcase : player.input
+      break if valid?(input)
+    end
```
Comment (margdaly) [inline lib/turn.rb:28]: I'm so happy this works again! Next time we'll get to use recursion.

**Review by margdaly — APPROVED** [review]
Thanks for the fixes! Looks great!

**Comment by grace-joh** [comment]
thank you for reviewing them! our game works now!! yay!

**Pull request merged**